phpBB/language/en/cli.php

```diff
@@ -126,12 +135,22 @@
 	'CLI_THUMBNAIL_NOTHING_TO_DELETE'	=> 'No thumbnails to delete.',
 
 	'CLI_USER_ADD_SUCCESS'		=> 'Successfully added user %s.',
+	'CLI_USER_DELETE_CONFIRM'	=> 'Are you sure you want to delete ‘%s’? [y/N]',
+	'CLI_USER_RECLEAN_START'	=> 'Re-cleaning usernames',
+	'CLI_USER_RECLEAN_DONE'		=> [
```

Comment:
No, we're using it in new code in phpBB quite a bit now
https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/di/container_builder.php#L93
https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/files/types/remote.php#L105-L109
https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/routing/file_locator.php#L24
https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/routing/loader_resolver.php#L26
https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/routing/resources_locator/default_resources_locator.php#L63
and others...